scripts/migrate.js

```diff
@@ -252,9 +272,9 @@ const q = faunadb.query;
     })
   );
 
-  // TODO: Show only the key rather than the entire result of the query with a message explaining you should include it in your .env file
-  console.log(guestKey);
+  console.log(`Please put this key in your .env.local file:`.bold.green, `NEXT_PUBLIC_FAUNA_SECRET=${guestKey.secret}`.bgBlack.white);
```

Comment:
No need to mention the .env.local or .env file name. The migration process also applies to production environments and thus the key may end up in an environment configuration like Vercel's or Netlify's rather than a file.